src/test/config.sol

```diff
@@ -106,7 +106,7 @@ contract Config {
         //
         afterSpell.line_offset           = 680 * MILLION;  // Offset between the global line against the sum of local lines
         afterSpell.pot_dsr               = 13_00;          // In basis points
-        afterSpell.pause_delay           = 16 hours;       // In seconds
+        afterSpell.pause_delay           = 30 hours;       // In seconds
         afterSpell.vow_wait              = 156 hours;      // In seconds
         afterSpell.vow_dump              = 250;            // In whole Dai units
         afterSpell.vow_sump              = 50 * THOUSAND;  // In whole Dai units
```

Comment:
nit, it's more of maintenance/improvements related so feel free to skip it from the spell.
I'd suggest to add `afterSpell.line_mom_authority` check in `config.sol` and `_checkSystemValues`. (only if it makes sense to add it in this spell)

Reply:
It's a simple addition and it makes sense. Good catch!